mindsdb/interfaces/jobs/README.md

```diff
@@ -0,0 +1,146 @@
+
+## Create job
+
+**Syntax**
+```sql
+CREATE JOB [<project_name>.]<job name> [AS] (
+   <mindsdb sql query>[; <mindsdb sql query>]
+)
+START { now | <date> }
+END { now | <date> }
+EVERY [<numer>] <period> 
+```
+Parameters:
+- START - optional, first date when job have to start. Default is now
+- END - optional, after this date job shouldn't run anymore. If not set - repeat forever
+- EVERY - optional job repeat rule.
+  - If not set - run once (no mather what was set in END)
+  - if <number> is not set - default value is 1 
+
+Supported periods:
+- minute/minutes/min
+- hour/hours
+- day/days
+- week/weeks
+- month/months
+
+Supported date formats:
+- '%Y-%m-%d %H:%M:%S'
+- '%Y-%m-%d'
+
+**Examples**:
+1:
+```sql
+create job proj1.retrain_rentals (
+   retrain mindsdb.home_rentals_model
+   using
+      join_learn_process=true;
+
+   insert into int1.rentals(
+      select * from int2.table1
+      join mindsdb.home_rentals_model
+   )
+) 
+end '2023-04-01 00:00:00'
+every 2 days
+```
+
+This example creates job in 'proj1' project and runs it every 2 days till 1st April. 
+
+This job have to
+- retrain predictor 
+- use trained predictor on records from table 'int2.table1'
+- append results of prediction to int1.rentals table
+
+join_learn_process=true - this parameter is used for waiting to retrain predictor
```

Comment:
Thinking at a higher level, maybe it's worth introducing a blocking mechanism of some kind for this? Because other jobs like adjust or create would also be async and hence we need to wait. Or maybe just using `join_learn_process` on all three of those is good enough for now.

Reply:
I think join_learn_process is enough for now. Because it should work for retrain and adjust too

Reply:
Agreed 👍